src/plugins/index.js

```diff
@@ -1 +1,2 @@
-export default {};
+// eslint-disable-next-line
+export { default as VirtualKeyboard } from './virtualKeyboardVueIntegration';
```

Comment:
```suggestion
export { default as VirtualKeyboard } from './virtualKeyboard';
```

Since we are in a vue project and in plugins I would rename this file.